@here/olp-sdk-dataservice-read/lib/client/CatalogVersionRequest.ts

```diff
@@ -25,6 +25,7 @@ import { validateBillingTag } from "..";
 export class CatalogVersionRequest {
     private startVersion?: number;
     private endVersion?: number;
+    private earliestVersion?: number;
```

Comment:
Please create separate request class for that method: CatalogEarliestVersionRequest